scripts/repo.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
Did you mean to include this file? I'm not sure it make sense to include this in our repository.

Reply:
Yes.  It's a script that's a collection of tools to make managing files in the repo easier.   E.g. moving headers.